src/logger.ts

```diff
@@ -43,4 +43,8 @@ const options: LoggerOptions = {
 const logger = createLogger(options);
 logger.info(`Logging initialized at ${logLevel} level`);
 
+export function buildNamedLog(message: string, name?: string): string {
+  return `${name ? `${name} - ` : ''}${message}`;
+}
```

Comment:
This nested string template thing is decidedly ugly, but I can't decide what the best way to re-write this is.

Here's some other ways I typed out but couldn't argue any of them are "better".

```ts
const prefix = name ? `${name} - ` : '';
return prefix+message;
```

```ts
return (name ? name + ' - ' : '') + message;
```

But the real winner is to allow any number of messages and concatenate them:

```ts
export function buildMessage(...messages: string[]): string => messages.join(' - ');
```

Reply:
Leaving a comment here for reference, will be refactoring the logger to allow all log statements to use the multi-message setup in a separate PR